app/api/socketio/standaloneEmitSocketEvent.ts

```diff
@@ -0,0 +1,23 @@
+import { Emitter } from '@socket.io/redis-emitter';
+import { createClient } from 'redis';
+import { config } from 'api/config';
+
+const emitSocketEvent = async (tenant: string, event: string, data: string) =>
+  new Promise((resolve, reject) => {
+    const redisClient = createClient({ host: config.redis.host, port: config.redis.port });
+
+    redisClient.on('error', reject);
+    redisClient.on('end', resolve);
+
+    redisClient.on('ready', () => {
+      const io = new Emitter(redisClient);
+      if (tenant === 'all') {
```

Comment:
This is susceptible to collide with a tenant named `all`: we have discussed and passing an INTENTIONAL empty string for `all` could be a better approach.

Reply:
done